sycl/test-e2e/Matrix/Inputs/joint_matrix_bfloat16_impl.hpp

```diff
@@ -114,35 +134,68 @@ int main() {
-      // test<bfloat16, bfloat16, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
-      test<bfloat16, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
-      // test<bfloat16, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
-      test<bfloat16, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
-      // test<bfloat16, bfloat16, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
-      // test<bfloat16, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
-      // test<bfloat16, bfloat16, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
-      // test<bfloat16, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
-      // test<bfloat16, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
+#if (!defined(ACC_BFLOAT16))
+      test<bfloat16, float, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, float, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
+#endif
+#if (defined(ACC_BFLOAT16))
+      // 8x16x16
+      test<bfloat16, bfloat16, float, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, float, bfloat16, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, bfloat16, bfloat16, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+
+      // 16x16x16
+      test<bfloat16, bfloat16, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, float, bfloat16, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, bfloat16, bfloat16, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
+
+      // 1x64x16
+      test<bfloat16, bfloat16, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, bfloat16, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
+
+      // 1x64x32
+      test<bfloat16, bfloat16, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, float, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, bfloat16, bfloat16, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, float, float, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
+
+      // 32x64x16
+      test<bfloat16, bfloat16, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, bfloat16, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, bfloat16, bfloat16, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
+      test<bfloat16, float, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
+
+      // 32x64x32
+      test<bfloat16, bfloat16, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, float, bfloat16, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, bfloat16, bfloat16, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
+      test<bfloat16, float, float, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
+#endif
```

Comment:
add macro name in comments, when multiple macros are used to make reading easier.
```suggestion
#endif // ACC_BFLOAT16
```